.github/NEW_IA_TEMPLATE.md

```diff
@@ -0,0 +1,49 @@
+# New {IA Name} Fathead Instant Answer
+Help DuckDuckGo become the best Search Engine for {Language} developers!
+
+
+### What do I need to know?
+You'll need to know how to code in **Perl**, **Python**, **Ruby**, or **JavaScript**.
+![fathead languages](https://cloud.githubusercontent.com/assets/873785/19787916/57b4c31a-9c73-11e6-9bd9-f85c8893ec93.jpg)
+
+
+### What am I doing?
+You will write a script that scrapes or downloads the data source below, and generates an **output.txt** file containing the parsed documentation.
+
+You can learn more about Fathead, and the `output.txt` syntax [**here**](https://docs.duckduckhack.com/resources/fathead-overview.html)!
```

Comment:
A minor thing but I'd suggest this:
You can learn more about Fatheads and the `output.txt` syntax [**here**](https://docs.duckduckhack.com/resources/fathead-overview.html).